pom.xml

```diff
@@ -545,7 +545,7 @@
             <plugin>
                 <groupId>org.owasp</groupId>
                 <artifactId>dependency-check-maven</artifactId>
-                <version>5.0.0-M3</version>
+                <version>5.0.0</version>
```

Comment:
You should disable the `Sonatype OSS Index Analyzer` cache because there is a bug in versions 5.0.0-M3 through 5.0.0. See https://github.com/jeremylong/DependencyCheck/issues/2016#issuecomment-505202284.

https://github.com/connexta/multi-int-store/pull/66/commits/a56db0bf90374bbf7a88fbc689e8493aebe1e71b

Reply:
This probably isn't relevant anymore. The OSS service was down when we experienced this issue.

Reply:
Seems like we should still disable that until we have an OSS cache configured for CI builds.